www/src/pages/404.astro

```diff
@@ -0,0 +1,60 @@
+---
+import LandingPage from "../layouts/landingPage.astro";
+import Button from "../components/landingPage/button.astro";
+---
+
+<LandingPage>
```

Comment:
There is some weird thing with the height of the page (at least on my phone) See: https://create-t3-90d36hrg1-t3-oss.vercel.app/fff